src/js/utm_params.js

```diff
@@ -0,0 +1,28 @@
+function generateUTMparams(type, pathname) {
+  
+  function generateUTMsource() {
```

Comment:
@codegoalie I should have put notes here. What I was trying to do was allow a place to put in one or many different paths. For example, if the marketing team says "I want blog post https://videofruit.com/blog/wrong-to-profit/ to pass utm_source of "posted-on-twitter-april-20". Then that logic could be added here as I was thinking there might be several requests to add custom utm_sources.

Reply:
I see. I think the official UTM params are not named very intuitively. I have to pull up the Wikipedia page whenever I work with them.

I think we want the `utm_source` to be pretty consistent based on where the link is placed, mainly in the nav.

Then the `utm_content` changes based on the contents of the page.

As such, I added 44b90ef as an example of what I was thinking. What do you think of that?

Reply:
I see. This makes sense. My process got into "hacking" a bit there.